s/nub-context/nub-context.ts

```diff
@@ -0,0 +1,70 @@
+import {dispatchCustomEvent} from "../nub-keyboard/utils/dispatchCustomEvent.js"
+import {keys} from "../nub-keyboard/utils/keys.js"
+import {styles} from "./style.css.js"
+
+export class NubContext extends HTMLElement {
+	shadow = this.attachShadow({
+		mode: "open",
+		delegatesFocus: false,
+	})
+
+	constructor() {
+		super()
+		let editable = false
+		this.shadow.innerHTML = `
+		<div>
+		<button class="flip">flip mode</button>
+		<slot>
+		</slot>
+		</div>
+		`
+		const style = document.createElement("style")
+		style.textContent = styles
+		this.shadowRoot?.append(style)
+		const flipButton = this.shadowRoot?.querySelector<HTMLElement>(".flip")
+		const nubContext = document.querySelector("nub-context")
+		nubContext?.addEventListener("nub-input", (e: any) => {
+			const detail = e.detail
+			if (detail?.actionName) {
+			dispatchCustomEvent(this, detail.actionName)
+			}
+		})
+		nubContext?.addEventListener("nub-action", (e: any) => {
```

Comment:
i don't think nub-context elements should be listening for nub_action events at all -- it's nub-context's job to *dispatch* nub_action events, not listen for them.